cargo-paradedb/README.md

```diff
@@ -43,9 +43,9 @@ Benchmark tools are run under the `cargo paradedb bench` subcommand, which are o
 cargo paradedb bench eslogs query-search-index
 ```
 
-### Benchmark pg_search
+### Benchmark pg_search, GIN-based tsvector indexes
```

Comment:
Let's make a different section for benchmarking tsvector please, instead of overwriting the `pg_search` section